site/frontend/.parcelrc

```diff
@@ -0,0 +1,9 @@
+{
+    "extends": "@parcel/config-default",
+    "compressors": {
+        "*.{html,css,js,svg,map}": [
```

Comment:
```suggestion
        "*.{css,js}": [
```
I wouldn't worry about compressing the `map` files, since we don't ship them to production anyway. We also don't have `.svg` nor `.html` files currently in the output of `parcel`.